doc/howto/instances_create.md

```diff
@@ -94,3 +94,29 @@ To launch a container with this instance type, enter the following command:
     lxc launch images:ubuntu/22.04 my-instance --type t2.micro
 
 The list of supported clouds and instance types can be found at [`https://github.com/dustinkirkland/instance-type`](https://github.com/dustinkirkland/instance-type).
+
+### Launch a VM that boots from an ISO
+
+To launch a VM that boots from an ISO, you first need to create a VM. In this scenario it might make sense to create an empty VM as the ISO image could be used for installation. An empty VM can be created using the following command:
+
+    lxc init iso-vm --empty --vm
+
+The second step it to import an ISO image which can later be attached to the VM as a storage volume:
+
+    lxc storage volume import path-to-image.iso iso-volume --type=iso
+
+Lastly, you need to attach the custom ISO volume to the VM using the following command:
+
+    lxc storage volume attach <pool_name> iso-volume iso-vm
+
+Now it's time to start the VM. By default the VM will try booting from its disk. For the VM to boot from the attached ISO, you need to be connected to the console console, and repeatedly press `ESC` until the firmware menu appears:
```

Comment:
```suggestion
Now it's time to start the VM. 
By default, the VM will try booting from its disk. 
To make the VM boot from the attached ISO instead, connect to the console and repeatedly press `ESC` until the firmware menu appears:
```

Reply:
@monstermunchkin have you tried making this easier by changing the boot.priority setting on the disk device when its attached by using `lxc config device add ` rather than `lxc storage volume attach`?